lib/src/string_extension.dart

```diff
@@ -96,3 +105,16 @@ extension PackageInfoExtension on String {
     }
   }
 }
+
+extension LocalizationExtension on String {
+  String get locale => this.tr();
```

Comment:
maybe it's have problem as user couldnt use easy localization and it will be not work, please back this code